### PR TITLE
In-scene placemarks

### PR DESCRIPTION
**Pull request by ronplaymeta**
Hi @pablo-mayrgundter, I updated codes for firs PR of https://github.com/bldrs-ai/Share/issues/599.
Please check the result and give me feedback.
Thank you.

### CODE REVIEW

**Comment by OlegMoshkovich** [comment]
@ronplaymeta the preview was failed - some tests are failing please fix.

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]
- Per convo, please remove any changes to Notes from this PR and submit those in a subsequent change (and probably after Oleg's work goes in).

- Please move what you can of the changes to CadView into a neighboring file.  My recent PR to introduce the SceneLayer is a good example.

- For the icon, please edit it to remove the attribution.  There should be only a couple paths in there for the main shape.  We should note the attribute though, so please add a README.md in the assets directory listing where it came from, with URL.

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]
About the placemark itself, it should always face towards the user. Can you try the sprite material as discussed in the dicord thread.

It also has a white background.  See if you can make this transparent.  The SVG file should just have paths for the shape, with no background.

**Comment by ronplaymeta** [comment]
ptal

File: src/utils/svg.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+      const material = new THREE.MeshBasicMaterial({map: texture, side: THREE.DoubleSide})
+      material.map.minFilter = THREE.LinearFilter
+      const mesh = new THREE.Mesh(geometry, material)
+      resolve(mesh)
+    }
+  })
+}
+
+
+export const getSVGSprite = async ({
```
Comment (pablo-mayrgundter) [inline src/utils/svg.js:156]: Please use function with jsdoc for top-level funcs

File: src/utils/svg.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  group.scale.y = scaleY
+  group.position.x = -width
+  group.position.y = height
+  group.scale.y *= - 1
+  svgGroup.add(group)
+  return svgGroup
+}
+
+
+export const getSVGMesh = async ({
```
Comment (pablo-mayrgundter) [inline src/utils/svg.js:119]: Please use function with jsdoc for top-level funcs

File: src/utils/svg.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+import debug from './debug'
+
+
+const svgLoadingManager = new THREE.LoadingManager()
+const svgLoader = new SVGLoader(svgLoadingManager)
+const fileLoadingManager = new THREE.LoadingManager()
+const fileLoader = new THREE.FileLoader(fileLoadingManager)
+
+
+export const getSVGGroup = async ({
```
Comment (pablo-mayrgundter) [inline src/utils/svg.js:13]: Please use function with jsdoc for top-level funcs

File: src/utils/svg.js [hunk 4]
```diff
@@ -0,0 +1,199 @@
+import * as THREE from 'three'
```
Comment (pablo-mayrgundter) [inline src/utils/svg.js:1]: Please add unit tests for this class

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]
Getting close.  Per convo, still need to remove the Notes work (for later) and please test the next svg code.

There's also some commented code.. if that's for debugging please remove.. tho realize it might be prep for follow-up work, in which case it's ok to leave.

File: src/utils/svg.js (hunk 1 above)
Comment (ronplaymeta) [reply src/utils/svg.js]: done

File: src/utils/svg.js (hunk 2 above)
Comment (ronplaymeta) [reply src/utils/svg.js]: done

File: src/utils/svg.js (hunk 3 above)
Comment (ronplaymeta) [reply src/utils/svg.js]: done

File: src/utils/svg.js (hunk 4 above)
Comment (ronplaymeta) [reply src/utils/svg.js]: done

**Comment by ronplaymeta** [comment]
ptal

**Comment by ronplaymeta** [comment]
ptal

**Comment by ronplaymeta** [comment]
ptal

**Comment by ronplaymeta** [comment]
ptal

**Comment by pablo-mayrgundter** [comment]
Heya, looks like you're still working...or is this ready for review?

**Comment by ronplaymeta** [comment]
It's ready to review, please check. :)
ptal

**Pull request merged**